components/infobox/wikis/pubgmobile/infobox_team_custom.lua

```diff
@@ -26,15 +26,19 @@ function CustomTeam.run(frame)
 end
 
 function CustomTeam:createBottomContent()
+	local upcomingTable = ''
+	if not _team.args.disbanded then
+		upcomingTable = upcomingTable .. Template.expandTemplate(
+			mw.getCurrentFrame(),
+			'Upcoming and ongoing tournaments of',
+			{team = _team.name or _team.pagename}
+		)
+	end
 	return Template.expandTemplate(
-		mw.getCurrentFrame(),
-		'Upcoming and ongoing tournaments of',
-		{team = _team.name or _team.pagename}
-	) ..  Template.expandTemplate(
 		mw.getCurrentFrame(),
 		'Placement summary',
-		{team = _team.name or _team.pagename}
```

Comment:
imo do not expand template, use the module it invokes directly

Reply:
Have invoked Placement summary directly
Upcoming tourna.... does not invoke a module atm, can be rewritten to do so in the future

Reply:
yeah i meant the `Placement summary`